wg/multiple-dc-enhancemant/README.md

```diff
@@ -0,0 +1,35 @@
+# multi-dc-enhancement Working Group
+
+The multi-dc-enhancement Working Group aims to implement an enhancement of deploying
+TiDB across multiple data centers.
+
+## Stakeholder SIGs
+
+List the stakeholder SIGs of this Working Group.
+
+## Meetings
+
+## Mentors
+
+* [Jay](https://github.com/busyjay), [PingCAP](https://github.com/pingcap)
+* [guoleitao](https://github.com/ltguo)
+
+## Organizers
+
+* [Peng Qu](https://github.com/hicqu)
+
+## Members
+
+* [haoxiang](https://github.com/haoxiang47)
```

Comment:
Please also unify the markdown usage of list - either use `*` or `-`. Not both in the same doc

Reply:
Any style guide?

Reply:
This is a common rule that may not be covered by specific guide - because the two symbols are actually displayed the same on website. For the source file, a consistent usage would be more friendly for writers or contributors.

Reply:
They are diplayed the same, so I think no constraints is better. Generally it's an obstacle for writers, you can take a look at https://github.com/tikv/community/blob/master/committee/wg-governance/working-group-template.md, seems more writers don't care it.

Reply:
Just a minor suggestion from writer perspective. Guess it does not much for this file.  IMO... maybe we should modify the template.